t/plugin/openwhisk.t

```diff
@@ -0,0 +1,209 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+no_long_string();
+no_root_location();
```

Comment:
New test file should contain https://github.com/apache/apisix/blob/1018576e30477a9d1e70710386aac998a68acc68/t/wasm/route.t#L31 to reduce repeated fields.

Reply:
Yes, it can help us reduce duplicate code. However, can we support this default setting globally instead of adding our own block processor in each `.t` file.

Reply:
The legacy code may be a problem. But you can give it a try.

Reply:
added